devicon.json

```
@@ -2934,6 +2934,28 @@
             }
         ]
     },
+    {
+        "name": "githubcodespaces",
+        "altnames": [],
+        "tags": [
+            "development",
+            "remote-development",
+            "editor",
+            "browser",
+            "cloud"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ],
+            "font": [
+                "plain"
+            ]
+        },
+        "color": "#ffffff",
```

Should be the dominant color of the icon unless anything else is mentioned in their branding guidelines
```suggestion
        "color": "#24292e",
```

---

I chose based on `plain` version, but yeah it makes more sense to base the choice on `original` version.
Committing changes!